### ISSUE TITLE
UK dialing codes

### ISSUE THREAD

**Issue by TwinMist** [brick/phonenumber#9]
Hi great bit of software, do you think you could build in UK dialing codes in to the parser?

https://www.ofcom.org.uk/phones-telecoms-and-internet/advice-for-consumers/advice/telephone-area-codes-tool

thanks

**Comment by BenMorel** [comment]
Hi, this library is just a wrapper for [giggsey/libphonenumber-for-php](https://github.com/giggsey/libphonenumber-for-php), so if support for specific calling codes is missing, this should be the place to ask for it.

I can open an issue with them though, can you please give me somes examples of what you're trying to parse, and what the expected output is?

**Comment by TwinMist** [comment]
Hi
Basically would like it to prase UK phone numbers and output the area based
on the dialing code and also if is a freephone, premium rate,  etc

Many thanks

**Comment by BenMorel** [comment]
OK, the upstream library does have a feature to extract the area code, when available.

I added a new method to support this: `PhoneNumber::getGeographicalAreaCode()`.
It's released in version [0.2.2](https://github.com/brick/phonenumber/releases/tag/0.2.2).

As for the phone number type, it's already available under `getNumberType()`.

Example:

```php
use Brick\PhoneNumber\PhoneNumber;
use Brick\PhoneNumber\PhoneNumberType;

$phoneNumber = PhoneNumber::parse('+442079460585');

echo $phoneNumber->getGeographicalAreaCode(); // 20

var_export($phoneNumber->getNumberType() === PhoneNumberType::FIXED_LINE); // true
```
